tests/models/report/test_validators.py

```diff
@@ -88,6 +89,22 @@ def test_validate_rml_sample(caplog):
         assert "The delivery report generation does not support RML samples" in caplog.text
 
 
+def test_validate_gender(caplog):
+    """Tests report gender parsing"""
+
+    # GIVEN an invalid gender category
+    gender = "female"
```

Comment:
```suggestion
    gender = Gender.FEMALE
```
From cg/constants/subject.py. These should be used as much as possible - so that we do not hard code